strings.txt

```diff
@@ -356,7 +356,7 @@ SLIMP3_HOME
 	ZH_CN	SLIMP3主页
 
 SQUEEZEBOX_HOME
-	CS	Squeezebox - Hlavní nabídka
```

Comment:
You changed your mind? This used to be "Doma" before you changed it last time?

Reply:
Yes, a little 😀 I've aligned it with the LMS Material